t/ui/10-tests_overview.t

```diff
@@ -139,7 +139,7 @@ is(scalar @filtered_out, 0, 'result filter correctly applied');
 # Test whether all URL parameter are passed correctly
 my $url_with_escaped_parameters
   = $baseurl
-  . 'tests/overview?arch=&flavor=&machine=&test=&modules=&module_re=&group_glob=&not_group_glob=&distri=opensuse&build=0091&version=Staging%3AI&groupid=1001';
+  . 'tests/overview?arch=&flavor=&machine=&test=&modules=&module_re=&group_glob=&not_group_glob=&comment=&distri=opensuse&build=0091&version=Staging%3AI&groupid=1001';
```

Comment:
Can/should we add an actual comment here or in an additional case? It looks like it would be trivial?

Reply:
Answering my own q. The logic is tested in the previously added test. Here we just confirm the arg is in the URL.